CHANGELOG.md

```diff
@@ -10,7 +10,13 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 The following sections document changes that have been released already:
 
-N/A
+### Bugfix
+
+#### browser and node
+
+- The `login` even was emitted before setting the session info, which
```

Comment:
```suggestion
- The `login` event was emitted before setting the session info, which
```

Though maybe there's not a need to go in so much detail - just a "session information sometimes wasn't available when `onLogin` was triggered". But either way :)

Reply:
Good point, I copy-pasted the commit message, but it actually makes sense that the changelog entry only summarizes the issue from a user perspective, and not details the internals of the bug.